.github/workflows/deploy-apis-to-production.yml

```diff
@@ -341,10 +341,10 @@ jobs:
           password: ${{ secrets.GCR_CONFIG }}
 
       - name: Build and push measurements source connectors
-        uses: docker/build-push-action@v2
+        uses: docker/build-push-action@v3
         with:
```

Comment:
[update the build push action version](https://github.com/airqo-platform/AirQo-api/pull/1216/commits/422ecde7829e79c9be18d84d0695a786aec6e00b)